### PR TITLE
Rns test suite

### PR DESCRIPTION
**Pull request by TheMarstonConnell**
Working through building proper unit tests for all of RNS

### CODE REVIEW

File: x/rns/types/utils.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+	if err != nil {
+		return "", "", err
+	}
+
+	return name, tld, nil
+}
+
+func GetCost(tld string) int64 {
+	cost := TLDCost[tld]
+	fmt.Printf("COST: %d\n", cost)
```
Comment (freak12techno) [inline x/rns/types/utils.go:64]: use logger instead?

Comment (faddat) [reply x/rns/types/utils.go]: That is the more conventional way

**Comment by faddat** [comment]
oooooooo

Great work 👏

**Review by faddat — APPROVED** [review]
looks good!

oh and that is such a pretty mockgen.sh file there.

**Pull request merged**